src/config/config.dev.ts

```diff
@@ -19,6 +19,9 @@ const DevConfig: QuaggaJSConfigObject = {
         singleChannel: false, // true: only the red color-channel is read
         debug: {
             showImageDetails: false, // logs frame grabber info, canvas size adjustments
+            showArea: false, // draws area boundaries on overlay when area is defined
```

Comment:
showArea should not be a "debug" parameter, it is of far more interest than just debugging.  It should be available to users without building a development build, as well as without enabling debug.  

Not sure if this would be beyond the scope of a "quick" change like we usually do from here, but the traditional way of implementing a visual on a barcode scanner, includes not only a thin border around the scan area, but as well, most of the rest of the image is dimmed out, and that makes the barcode that much much more.